en/docs/learn/non-functional-requirements.md

```diff
@@ -0,0 +1,31 @@
+## Availability Requirements
```

Comment:
Let's add a brief intro what non-functional requirements are and what this page explains.

Reply:
Also shall we rewrite this page to reduce complexity from the CDS spec.